mix.exs

```diff
@@ -36,7 +36,7 @@ defmodule EctoStateMachine.Mixfile do
 
   defp deps do
     [
-     {:ecto, ">= 1.1.2 or >= 2.0.0"},
+     {:ecto, ">= 1.0.0 or >= 2.0.0"},
```

Comment:
I have to use it with Ecto 1.0.0 (Mongo.Ecto currently 1.0.0 only)
Later you can release 2.0 only release.